test/integration/seed_modifiers.dbtspec

```diff
@@ -24,6 +24,7 @@ projects:
         seeds:
           project_table_modifiers_with_seed_table_kind:
             table_kind: "SET"
+            index: "UNIQUE PRIMARY INDEX (id)"
```

Comment:
@septimit, can you please explain why this is required now?

Reply:
+1